test/test_sediments.jl

```diff
@@ -59,13 +66,15 @@ total_nitrogen(::LOBSTER, model) = sum(Array(interior(model.tracers.NO₃))) +
                                    sum(Array(interior(model.tracers.DOM))) +
                                    sum(Array(interior(model.tracers.sPOM))) +
                                    sum(Array(interior(model.tracers.bPOM)))
+                                   
 total_nitrogen(::lobster_variable_redfield, model) = sum(Array(interior(model.tracers.NO₃))) +
                                                      sum(Array(interior(model.tracers.NH₄))) +
                                                      sum(Array(interior(model.tracers.P))) +
                                                      sum(Array(interior(model.tracers.Z))) +
                                                      sum(Array(interior(model.tracers.DON))) +
                                                      sum(Array(interior(model.tracers.sPON))) +
                                                      sum(Array(interior(model.tracers.bPON)))
```

Comment:
`sum(model.tracers.NO₃)` should work. It's preferred because it will omit immersed boundary regions, whereas writing `Array(interior(field))` will not.

Reply:
`sum(interior(field))` also works on the GPU and avoids the slow process of transferring data to the CPU.

Reply:
https://github.com/OceanBioME/OceanBioME.jl/blob/ac8419a1ce5a06a82be31588583636b212c09598/src/Models/AdvectedPopulations/LOBSTER/LOBSTER.jl#L453-L456

it's a collection of types; is `LobsterVariableRedfield` the suggested convention?

Reply:
Makes sense! Also Oceananigans style uses "english" semantics, so I think the type should be pronounced "VariableRedfieldLobster" as in "the variable-redfield lobster model"